ocaml/xapi/extauth_plugin_ADpbis.ml

```diff
@@ -27,6 +27,12 @@ let with_lock = Xapi_stdext_threads.Threadext.Mutex.execute
 let lwsmd_service = "lwsmd"
 
 module Lwsmd = struct
+  (* This can be refined by Mtime.Span.hour when mtime is updated to 1.4.0 *)
```

Comment:
Mtime in xs-opam is at 1.4.0, which as a package is at 6.59.0. This package is in rt-next. So I see no reason to not use 1.4.0.

Reply:
we need to back port this commit to yangtze  (as you see this update is for pbis, and pbis is not in rt-next),
I fix this in master and then back port to yangtze to keep consistency. 
I see yangtze still use 1.2.0.